unit_test/SequentialCatchMain.cpp

```diff
@@ -26,12 +26,14 @@
 
 #define CATCH_CONFIG_RUNNER
 #include <catch2/catch.hpp>
+#include <lbann/utils/dnn_lib/helpers.hpp>
 #include <lbann/utils/random_number_generators.hpp>
 
 int main(int argc, char* argv[]) {
-#ifdef LBANN_HAS_GPU
+#ifdef LBANN_HAS_DNN_LIB
   hydrogen::gpu::Initialize();
```

Comment:
We should probably explicitly call `dnn_lib::initialize()`, too, for consistency if nothing else.
```suggestion
  hydrogen::gpu::Initialize();
  dnn_lib::initialize();
```